File: src/ProjectR.Scripting/content/scripts/spells/WindGodTornado.lua
function GetTargetType()
  return Enemies
end

function GetName()
  return "Wind God Tornado"
end

function GetDescription()
  return "Summon a powerful tornado to wreak havoc among your enemies."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return 1.48
end

function GetDelay()
  return .2
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return { WND }
end

function SpellEffect()
  local damage = (6 * aMD * (aWND/100) - 1.5 * dMR) * (100/dWND)
  d:TakeDamage(damage)
end

File: src/ProjectR.Scripting/content/scripts/spells/WindGodTornado.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class WindGodTornado : SpellScriptBase
    {
        public override string Name { get { return "Wind God Tornado"; } }
        public override string Description { get { return "Summon a powerful tornado to wreak havoc among your enemies."; } }

        public override TargetType TargetType { get { return TargetType.Enemies; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { EleMastery.WND, }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 148; } }
        public override double Delay { get { return .2; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            var damage = (6 * aMD * (aWND / 100) - 1.5 * dMR) * (100 / dWND);
            DealDamage(damage);
        }
    }
}